doc/modules/mod_muc_light_commands.md

```diff
@@ -0,0 +1,49 @@
+# MongooseIM's multi-user chat light commands set
+
+## Purpose
+This is a set of commands, providing actions related to multi-user chat light features.
+It is currently used to expose the part of REST API responsible for managing rooms.
```

Comment:
It sounds like `mod_muc_light_commands` is exposing something from REST API but it is the other way round.